xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingMarketDataService.java

```diff
@@ -70,7 +84,8 @@ public BinanceStreamingMarketDataService(BinanceStreamingService service) {
 
     @Override
     public Observable<Trade> getTrades(CurrencyPair currencyPair, Object... args) {
-        throw new NotYetImplementedForExchangeException();
+      return getRawTrades(currencyPair)
+            .map(at -> new Trade(BinanceAdapters.convertType(at.buyerMaker), at.quantity, currencyPair, at.price, at.getTimestamp(), Long.toString(at.aggregateTradeId)));
```

Comment:
same as above this will cause confusion if the standard trade stream is implemented

Reply:
I have no problem implementing the "getRawAggTrades" as you suggested above. But I still think the override of:
```Observable<Trade> getTrades(CurrencyPair currencyPair, Object... args);```
Should return the aggregated trades here (Since we have to pick on if we implement both).

Maybe I'm missing something:
If we're not exposing (individual) trade ids , what's the point emitting single trades here ?